shell/platform/windows/angle_surface_manager.cc

```diff
@@ -126,13 +126,36 @@ bool AngleSurfaceManager::Initialize() {
     return false;
   }
 
+  egl_resource_context_ = eglCreateContext(
+      egl_display_, egl_config_, egl_context_, display_context_attributes);
+
+  if (egl_resource_context_ == EGL_NO_CONTEXT) {
+    OutputDebugString(L"EGL: Failed to create EGL resource context");
+    return false;
+  }
+
   return true;
 }
 
 void AngleSurfaceManager::CleanUp() {
+  EGLBoolean result = EGL_FALSE;
+
   if (egl_display_ != EGL_NO_DISPLAY && egl_context_ != EGL_NO_CONTEXT) {
-    eglDestroyContext(egl_display_, egl_context_);
+    result = eglDestroyContext(egl_display_, egl_context_);
     egl_context_ = EGL_NO_CONTEXT;
+
+    if (result == EGL_FALSE) {
+      OutputDebugString(L"EGL: Failed to destroy context");
+    }
+  }
+
+  if (egl_display_ != EGL_NO_DISPLAY && egl_context_ != EGL_NO_CONTEXT) {
```

Comment:
Should this be `egl_resource_context_`?

Reply:
Pushed a fix for this, since it's pretty clearly a copy/paste mistake.